include/circt/Dialect/FIRRTL/FIRRTLExpressions.td

```diff
@@ -1136,6 +1136,24 @@ def PathOp : FIRRTLOp<"path", [Pure,
   let assemblyFormat = "$target attr-dict";
 }
 
+def ListCreateOp : FIRRTLOp<"listcreate", [Pure]> {
```

Comment:
Can we use `SameTypeOperands` and remove `all_equal` in verification code?

Reply:
Of course, good call thanks!